nixos/modules/flyingcircus/services/prometheus/default.nix

```diff
@@ -67,7 +67,7 @@ let
         '';
       };
 
-      labels = mkOption {
+      external_labels = mkOption {
```

Comment:
This isn't referenced anywhere else. On purpose? Is this included transparently somewhere?

Reply:
This is going directly to prometheus configuration. And the option was renamed upstream.